### DISCUSSION TITLE
Searching existing Quasar forums

### DISCUSSION THREAD

**Discussion by maggie44** [quasarframework/quasar#8858] [General - Components / Directives / etc]
It seems like the old forums have a wealth of information in them, an archive that may well hold a lot of answers for me (and spare crowding out the discussions pages). But sign ups are disabled and you can’t search without signing up. Can we allow searching without sign ups to make that wealth of info available? 

https://forum.quasar-framework.org/

**Comment by ontwikkelfabriek** [comment]
As a workaround you could use google to search the forum like:

[example to search quasar forum on 'ssr' ](https://www.google.com/search?q=site%3Ahttps%3A%2F%2Fforum.quasar-framework.org+ssr&pws=0&gl=us&sxsrf=ALeKk02M76EPdyJwcptuZjZXbtfnT0HDHA%3A1618583471333&ei=r595YLbGE-CE9u8P9KizgAY&oq=site%3Ahttps%3A%2F%2Fforum.quasar-framework.org+ssr&gs_lcp=Cgdnd3Mtd2l6EAM6BAgjECc6BAguEEM6BAgAEEM6CgguEMcBEKMCEEM6BQgAEJECOggILhDHARCjAjoCCC46AggAOggILhDHARCvAVCHJliYngFg4K8BaAFwAngAgAGDAYgB_AiSAQM5LjSYAQCgAQGgAQKqAQdnd3Mtd2l6wAEB&sclient=gws-wiz&ved=0ahUKEwj22eOm_YLwAhVggv0HHXTUDGAQ4dUDCA4&uact=5)

**Comment by metalsadman** [comment]
I find it easier to just google search (in fact i haven't used the old forum's search function much), seo will still get you results from the old forums.

Just ask your question here, if you don't find a good result from googling. I would still link old forum topic if it was already answered before in there.
